contrib/platform/src/com/sun/jna/platform/win32/COM/WbemcliUtil.java

```diff
@@ -341,7 +346,13 @@ public void setWmiClassName(String wmiClassName) {
                             break;
                         // Unimplemented type. User must cast
                         default:
-                            values.add(vtType, cimType, property, pVal.getValue());
+                            if(((vtType & VT_ARRAY) == VT_ARRAY) ||
```

Comment:
I think we need to handle (as a case before the default) `VT_EMPTY` unless we're sure it will return `VT_ARRAY|VT_EMPTY`.  Might also want to throw in `VT_NULL` (return `null`).

Reply:
`VT_NULL` was already handled, I added `VT_EMPTY` to be handled the same way